app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
+        //export the font to a temporary directory
+        var temp_dir = instanceName+"_temp";
+        this.exportInstance(masterName, temp_dir, precision);
```

Comment:
I suggest we merge this branch into master and then we can open an issue for making this specific improvement to the implementation. It is not straightforward to implement, and I think it deserves some experimentation on its own. So let's not include this InMemory feature in this patch-set.